README.md

```diff
@@ -17,13 +49,32 @@ started.
 2. `node-red`
 3. Open <http://localhost:1880>
 
-## Getting Help
-
-More documentation can be found [here](https://nodered.org/docs).
-
-For further help, or general discussion, please use the [Node-RED Forum](https://discourse.nodered.org) or [slack team](https://nodered.org/slack).
-
-## Developers
+<br/>
+
+> [!NOTE]
+> More documentation can be found [here](https://nodered.org/docs). For further help, or general discussion, please join the [Node-RED Forum](https://discourse.nodered.org) or [Node-RED Slack](https://nodered.org/slack).
+
+## Links
+
+- [Documentation ↗](https://nodered.org/docs/)
+- [About ↗](https://nodered.org/about/)
+- [Installation ↗](https://nodered.org/#get-started)
+- Roadmap (Coming soon)
```

Comment:
Slightly wary of adding lots of 'coming soon' indicators for content we don't current have a plan for. If they still say 'coming soon' a year from now it isn't a good look.

Reply:
@knolleary agreed. I added them in here to throw a ball towards what we can think of in the future to include. I'll remove them for now..